app/models/import_batch.rb

```diff
@@ -38,7 +38,13 @@ def create_entries
     existing_mappings = site.mappings.where(path: canonical_paths)
 
     deduplicated_csv_rows.each do |row|
-      entry = ImportBatchEntry.new(path: row.path, type: row.type, new_url: row.new_url)
+      entry = ImportBatchEntry.new(
+        path: row.path,
+        type: row.type,
+        new_url: row.new_url,
+        archive_url: row.archive_url,
```

Comment:
Re your questions in the [commit message](https://github.com/alphagov/transition/commit/59ddcab0bb384e20c390e2acf64db994cc33c864) about validations on the batch, we should probably validate the archive URLs for length, but there's no point in validating for non-blankness or that they're National Archives URLs. They will only be set as archive URLs on the entries if their [hosts matched `NationalArchivesURLValidator:: NATIONAL_ARCHIVES_HOST `](https://github.com/alphagov/transition/blob/4f2fe54d6753e4fcff11390c7ba8f91a75a1665e/lib/transition/import/csv/import_batch_row.rb#L73) which covers both of those cases already.